cmd/stacker/internal_go.go

```diff
@@ -208,3 +224,80 @@ func doAtomfsUmount(ctx *cli.Context) error {
 	mountpoint := ctx.Args().Get(0)
 	return atomfs.Umount(mountpoint)
 }
+
+func doBomDiscover(ctx *cli.Context) error {
+	author := "stacker-internal"
+	org := "stacker-internal"
+
+	if err := fs.Discover(author, org, "/stacker/artifacts/installed-packages.json"); err != nil {
+		return nil
+	}
+
+	return nil
+}
+
+func doBomGenerate(ctx *cli.Context) error { //nolint:unused // used when invoked inside "run:"
+	if ctx.Args().Len() != 1 {
+		return errors.Errorf("wrong number of args for umount")
+	}
+
+	input := ctx.Args().Get(0)
+
+	author := "stacker-internal"
+	org := "stacker-internal"
+	lic := "unknown"
+
+	if err := distro.ParsePackage(input, author, org, lic, fmt.Sprintf("/stacker/artifacts/%s.json", filepath.Base(input))); err != nil {
+		return nil
+	}
+
+	return nil
+}
+
+// pkgname, license, paths...
```

Comment:
improve or remove this comment.  It doesn't line up with the Args().Get calls below.

Reply:
Fixed